include/vere/vere.h

```diff
@@ -600,6 +600,7 @@
         c3_i       xit_i;                   //  exit code for shutdown
         void*      tls_u;                   //  server SSL_CTX*
         u3_trac    tra_u;                   //  tracing information
+        void     (*onboot)();               //  call when chis is up
```

Comment:
I think we should keep the naming scheme for now, in which case this is `bot_f` or something else with three letters and then `_f` for function pointer.